src/test/java/seedu/address/model/policy/PolicyListTest.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.policy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalPolicies.LIFE_TIME;
+import static seedu.address.testutil.TypicalPolicies.LIFE_TIME_NAME;
+import static seedu.address.testutil.TypicalPolicies.MISSING_FROM_LIST_POLICY;
+import static seedu.address.testutil.TypicalPolicies.MISSING_FROM_LIST_POLICY_NAME;
+import static seedu.address.testutil.TypicalPolicies.getTypicalPolicyList;
+
+import java.util.Hashtable;
+
+import org.junit.jupiter.api.Test;
+
+public class PolicyListTest {
+
+
+    @Test
+    public void constructor() {
+        PolicyList policyList = new PolicyList();
```

Comment:
Perhaps we should have standardised names for the tests. Same for the rest of the tests in this file